STUDIO/org.openl.rules.test/test-resources/functionality/EPBDS-14557_Vocabulary_Validation.xlsx.msg.txt

```diff
@@ -0,0 +1,15 @@
+Object 'F1' is outside of valid domain 'Gender'. Valid values: [Male, Female]
+Object '2' is outside of valid domain 'Level'. Valid values: [100, 200, 300]
```

Comment:
Add test case like:
<table>
<tr><th>Datatype County &lt;String[]></th></tr>
<tr><td>NY, NJ</td></tr>
<tr><td>CA</td></tr>
</table>

<table>
<tr><th colspan=2>Spreadsheet SpreadsheetResult check(County county)</th></tr>
<tr><td>Step</td><td>Formula</td></tr>
<tr><td>Result</td><td>= county</td></tr>
</table>

and calls
`= check(new String[]{"CA"})` - valid
`= check(new String[]{"NY", "NJ"})` - valid
`= check(new String[]{"NJ", "NY"})` - ERROR
`= check(new String[][]{{"CA"}})` - valid
`= check(new String[][]{{"CA"}, {"NY", "NJ"}})` - valid
`= check(new String[][]{null, {"NY", "NJ"}})` - valid
`= check(new String[]{null})` - ERROR
`= check(null)` - valid

Reply:
= check(new String[]{null}) - ERROR 
= check(null) - valid
These scenarios can be either valid or erroneous right?